lib/cisco_node_utils/radius_server.rb

```diff
@@ -202,6 +298,7 @@ def default_accounting
     end
 
     def accounting=(val)
+      return if platform == :ios_xr
```

Comment:
Please remove this (and the corresponding line in `authentication=`. Since you have `_exclude`d the `accounting` property from ios_xr, `config_set` will raise an `UnsupportedError` which is the desired behavior.